packages/media-sound/cava/cava-0.10.0.exheres-0

```diff
@@ -37,11 +37,20 @@ DEFAULT_SRC_CONFIGURE_OPTION_ENABLES=(
     "pulseaudio input-pulse"
     "portaudio input-portaudio"
     "sdl output-sdl"
+    "sdl output-sdl-glsl"
     "sndio input-sndio"
 )
 
+DEFAULT_SRC_CONFIGURE_PARAMS=(
```

Comment:
Pipewire support has been added via `--disable/disable-input-pipewire` and is currently automagic  checking for pipewire >= 0.3 by:
`PKG_CHECK_MODULES(PIPEWIRE, libpipewire-0.3, have_pipewire=yes, have_pipewire=no)`
and should also be added. This also requires `virtual/pkg-config` as `build:` dep.


`--enable/disable-cava-font` configure switch has also been added.

General note: You can also submit MRs to ::media-unofficial on GitLab: https://gitlab.exherbo.org/exherbo-unofficial/media-unofficial/

Reply:
as far as I can see virtual/pkg-config is currently only used for detecting pipewire so it would be more correct to also hide that behind the pipewire? option requirement